embedding/embedlite/embedthread/EmbedLiteAppThreadParent.cpp

```diff
@@ -40,7 +40,6 @@ EmbedLiteAppThreadParent::RecvInitialized()
 {
   LOGT();
   PR_SetEnv("MOZ_USE_OMTC=1");
-  PR_SetEnv("MOZ_LAYERS_PREFER_OFFSCREEN=1");
```

Comment:
In theory if MOZ_USE_EXTERNAL_WINDOW=1 and HasGLContext == true, we would endup with creating CreateWrappingExisting, and no need to disable here offscreen context.